app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+    use Rack::Flash
+    get '/tweets' do
+        if logged_in?
+            @user = User.find_by_id(session[:user_id])
+            erb :'/tweets/tweets'
+        else
+            redirect to "/login"
+        end
+    end
+
+    get '/tweets/new' do
+        if logged_in?
+            erb :'/tweets/create_tweet'
+        else
+            redirect "/login"
+        end
+    end
+
+    post '/tweets' do
+        if !params[:content].empty?
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.